CONTRIBUTING.md

```diff
@@ -1,78 +1,324 @@
+    - ["2.0.0+"     , "using/. . ."]           #       -- Optional
+aliases:                                       # 9     -- Optional
+    - /riakts/1.3.1/. . .                      #       -- Optional
+canonical_link: "https://docs.basho.com/. . ." # 10    -- Conditional
 ---
 ```
 
-There is a separate layout for pages dedicated to presenting downloads links. There are a few additional elements that must be included, **in addition** to the above.
+1.  The Page Title is the title of the page, in the context of both the
+    &lt;head>&lt;title> element and as rendered on the page.
+
+    The rendered title will be placed above the primary content (above the text
+    that makes up the markdown document) as the only &lt;h1> element on the
+    page. If the Title Super-Text (1.b, `title_supertext`) is present, it will
+    be rendered above the Page Title, using a slightly reduced font.
+
+2.  The Description should be a quick, single-sentence summary of the page.
+
+    It will be used in a `<meta name="description">` tag for rich content
+    summaries and potentially as the title text that appears when hovering over
+    menu item elements.
+
+3.  The Project Designation that this page is associated with.
+
+    In the case of pages located under riak/kv/ and riak/ts, the designation
+    will use underscores, ex. `riak_kv`, and `riak_ts`.
+
+4.  The Project Version that this page is associated with.
+
+5.  "Hugo has a simple yet powerful menu system that permits content to be
+    placed in menus with a good degree of control without a lot of work."
+
+    https://gohugo.io/extras/menus
+
+    Our implementation of Hugo menus requires adherence to a few additional
+    conventions detailed below. For general information, please see the
+    documentation linked above
+
+    1.  The name of the menu for the page.
+
+        This name __must be__ `"${project}-${project_version}"`. The content
+        navigation generation logic assumes this to be true and will not be able
+        to correctly render our side bar if this is not the case.
+
+    2.  Abbreviated name of the page.
+
+        This text will be rendered as the page title in the content navigation
+        menu, and so length should be considered.
+
+    3.  Unique page identifier.
+
+        Hugo menus use these `identifier`s internally, and will assume that each
+        `identifier` is unique _to the menu_. Every version of the `riak_ts`
+        project can include the `using` identifier -- as each version builds its
+        own menu -- but each menu **must not** include duplicate entries.
+
+        Errors will be thrown at Hugo build time.
+
+    4.  The ordering weight of the page, relative to pages sharing a `parent`.
+
+        Order is ascending, which means the lowest (lightest) weight will be at
+        the top, and the highest (heaviest) will sink to the bottom. In case of
+        ties, alphabetical sorting of page `identifier` is used.
```

Comment:
I would like to add here: 

Reordering should only happen if you are adding a page. Otherwise, you should speak to someone in Docs before making any changes to the weights of pages.